index.js

```diff
@@ -3,7 +3,7 @@ const core = require('@actions/core');
 try {
   // Get inputs
   const input1 = parseInt(core.getInput('input1'));
-  const input2 = parseInt(core.getInput('input2'));
```

Comment:
There appears to be a typo in the name of the `input2` variable. It is being retrieved as `'inpt2'` instead of `'input2'`. This will likely cause an error or incorrect behavior. Please double-check and fix the variable name.